### PR TITLE
Updates `WMPopover` to include placement. Updates styling.

### PR DESCRIPTION
**Pull request by dr15**
See title

### CODE REVIEW

File: src/components/common/WMPopover/index.tsx [hunk 1]
```diff
@@ -1,18 +1,43 @@
...
+  LeftTop = 'leftTop',
+  Left = 'left',
+  LeftBottom = 'leftBottom',
+  RightTop = 'rightTop',
+  Right = 'right',
+  RightBottom = 'rightBottom',
+  BottomLeft = 'bottomLeft',
+  Bottom = 'bottom',
+  BottomRight = 'bottomRight',
+}
```
Comment (RonnyO) [inline src/components/common/WMPopover/index.tsx:20]: No need for this.
Use `import { TooltipPlacement } from 'antd/lib/tooltip';` as you just pass that through (and change the default placement to `'top'`.

Comment (dr15) [reply src/components/common/WMPopover/index.tsx:20]: `TooltipPlacement` is a list of string literals, I thought an enum would be more declarative and also this is something that was already done elsewhere and the project so I thought I'll go with it. I can remove this if you'd prefer that.

Comment (RonnyO) [reply src/components/common/WMPopover/index.tsx:20]: I don't see the same interface elsewhere, or why it's better this way. If the extra WM abstraction doesn't add anything, I wouldn't change types without a solid reason. The basic expectation of any WM component that wraps antd should be that most props are either passed untouched or a default is changed here and there, not that interfaces change.

Comment (dr15) [reply src/components/common/WMPopover/index.tsx:20]: I'll remove it when I get the chance

**Pull request merged**